shogun-lib/src/main/java/de/terrestris/shogun/lib/model/jsonb/layer/DefaultLayerSourceConfig.java

```diff
@@ -96,4 +96,10 @@ public class DefaultLayerSourceConfig implements LayerSourceConfig {
         example = "false"
     )
     private Boolean useBearerToken;
+
+    @Schema(
+        description = "Dimension parameters to be replaced for WMTS requests",
+        example = "{\"Time\": \"2010\"}"
+    )
+    private HashMap<String, String> wmtsDimensions;
```

Comment:
For `requestParams` the type is `HashMap<String, Object>`. I suppose this is not a problem?

Reply:
Technically it shouldn't be a problem, but I assume users must pass a string value (in the admin client)? If so it might be confusing since it's possible for the `requestParams`.